R/model.R

```diff
@@ -146,220 +160,368 @@ fall_run_model <- function(scenario = NULL, seeds = NULL){
                                             migratory_survival_bay_delta = migratory_survival$bay_delta,
                                             juveniles_at_chipps = juveniles_at_chipps,
                                             growth_rates = growth_rates)
-
-
+        
+        
         migrants_at_golden_gate <- delta_fish$migrants_at_golden_gate
-
+        
         annual_migrants <- annual_migrants + migrants_at_golden_gate
+        
       } else {
+        
+        if (month == 11 & year > 1) {
+          yearlings <- yearling_growth(year, round(yearlings))
```

Comment:
For `yearling_growth()` should we be using `yearling_dynamics_year()`? This one I am a little confused about but it looks like in the old model they do use the yr+1 in this case.